src/more-infos/more-info-alarm_control_panel.js

```diff
@@ -34,17 +34,44 @@ export default new Polymer({
       type: Boolean,
       value: false,
     },
+    code_input_visible: {
+      type: Boolean,
+      value: false,
+    },
+    code_input_enabled: {
+      type: Boolean,
+      value: false,
+    },
+    code_format: {
+      type: String,
+      value: '',
+    },
+    code_valid: {
+      type: Boolean,
+      value: false,
+    },
```

Comment:
Polymer allows you to have computed properties based on other properties. [See this example.](https://github.com/balloob/home-assistant-polymer/blob/master/src/components/entity/ha-entity-marker.js#L22)

```javascript
code_valid: {
  type: Boolean,
  computed: 'validate_code(entered_code)',
}
```